starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1586,15 +1584,22 @@ sub check_star_catalog {
             }
         }
 
-        # ACA-041
 	if ($type =~ /BOT|GUI|ACQ/){
 	    if (( $maxmag =~ /---/) or ($mag =~ /---/)){
 		push @warn, sprintf "[%2d] Magnitude.  MAG or MAGMAX not defined \n",$i;
 	    }
 	    else{
-		if (($maxmag - $mag) < $mag_faint_slot_diff){
-		    my $slot_diff = $maxmag - $mag;
-		    push @warn, sprintf "[%2d] Magnitude.  MAXMAG - MAG = %1.2f < $mag_faint_slot_diff \n",$i,$slot_diff;
+                # This is an explicit check of ACA-041
+                if (($maxmag - $mag) < 0.3){
+                    push @warn, sprintf "[%2d] Magnitude.  MAXMAG - MAG < 0.3\n", $i;
+                }
+                # And this is a sanity check on the current maxmag clipping at 11.2.
+                my $rounded_maxmag = sprintf("%.2f", $maxmag);
+                my $expected_maxmag = min($mag + 1.5, 11.2);
```

Comment:
It looks like this is throwing false positives (I think out of the matlab tools for larger mag errors, the original maxmag value would have been more than $mag + 1.5).

Reply:
That's an interesting question if this should be considered a False positive.  I.e. do we actually want to allow `maxmag > mag + 1.5` even for stars with large mag err?  My first reaction is that we should just stick with 1.5 no matter what.  Of course MATLAB is doing what it does now, so this warning serves as a notice of a large maxmag.  Remember that strictly speaking, starcheck warnings are evaluated by the ACA reviewer and a catalog can be passed at ACA reviewer discretion if it does not violate any FDB-controlled checklist items.

Reply:
Regarding "false positive", I just didn't think we wanted to add a new check that would add possibly a bunch of orange warning that we aren't going to care about.  For example,  For example star 1039018944 in 48042 looks to be a color=1.5 star that is being handled with a larger mag error on the matlab side and ends up with a 11.20 clipped value.  I thought the "green" handling of it as a 1.5 star was probably enough:

ttps://icxc.cfa.harvard.edu/mp/mplogs/2019/JUL0719/oflsa/starcheck.html#obsid48042

and it does not need 

>> WARNING : [ 6] Magnitude.  MAXMAG 11.20 not within 0.1 mag of 10.28

Reply:
Fair enough.  My point is that MATLAB should not be giving that star a maxmag that is so large.  Previously that was just fine and we didn't care, but now that could actually negatively impact acquisition probability (compared to the alternative of maxmag = mag + 1.5).  

Some visibility into this might be OK as long as it is not happening *too often*.  (No, I don't have a strict definition of what that means).  But I also agree (and have a long history of advocating) that non-actionable warnings should be avoided.  Can you make it an info?  I think we should be working toward changing MATLAB handling of maxmag, but that may take some consideration of cases like this.

Reply:
I agree on prioritizing figuring out what to actually do with the handling of maxmag .  And with regard to the warning coming up, it looks like it will just do that a lot more when we have more color=1.5 stars (which are often whole fields).  With regard to this new warning, sure I can go with blue / INFO or green warning / CAUTION.

Reply:
OK, I've set to green / CAUTION (consistent with the color of the row for the color=1.5 stars anyway).